report_qweb_parameter/models/ir_qweb.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2017 Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class IrQWeb(models.AbstractModel):
+    _inherit = "ir.qweb"
+
+    def check_length(self, value, min_length=False, max_length=False):  # noqa
+        """No use staticmethod because self is needed to translate exception messages"""
+        if min_length and len(value) < min_length:
+            raise ValidationError(_("Length cannot be less than %s") % str(min_length))
```

Comment:
```suggestion
            raise ValidationError(self.env._("Length cannot be less than %s") % str(min_length))
```
could you apply this https://github.com/odoo/odoo/commit/b794f0f332f473deb2c04eba60baf4761db3b508

Reply:
done